respy/pre_processing/model_processing.py

```diff
@@ -31,9 +31,7 @@
 
```

Comment:
Could you please restore the whole module? You must have accidentally deleted newlines throughout the document. Use ``git restore -s master -- respy/pre_processing/model_processing.py``.

Reply:
Not resolved.